packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/ValueSchemas.ts

```diff
@@ -170,6 +173,73 @@ const multipleConfigSchema = string()
     }
   );
 
+const singleInterfaceSchema = string()
+  .max(100, LENGTH_ERROR_MESSAGE)
+  .test(
+    'validate-single-interface-schema',
+    INTERFACE_ID_ERROR_MESSAGE,
+    function (value) {
+      if (!value || typeof value !== 'string') {
+        return this.createError({ message: fieldErrorMessage });
+      }
```

Comment:
do we have to show error on empty string as well? because empty value will be interpreted as false in boolean and !value make it throw error
```suggestion
      if (value===null || value === undefined || typeof value !== 'string') {
        return this.createError({ message: fieldErrorMessage });
      }
```

check for null and undefined accordingly if those are possible values

Reply:
@nikhagra-akamai We do want the error to be shown when user clicks on a component and not pass any value.